docs/mmctl_channel_rename.md

```diff
@@ -4,23 +4,26 @@ Rename a channel
 
 ### Synopsis
 
-Rename a channel.
+Rename an existing channel.
 
 ```
-mmctl channel rename [flags]
+mmctl channel rename [channel] [flags]
 ```
 
 ### Examples
 
 ```
-  channel rename myteam:mychannel newchannelname --display_name "New Display Name"
+  channel rename myteam:oldchannel --name 'new-channel' --display_name 'New Display Name'
+  channel rename myteam:oldchannel --name 'new-channel'
+  channel rename myteam:oldchannel --display_name 'New Display Name'
 ```
 
 ### Options
 
 ```
       --display_name string   Channel Display Name
   -h, --help                  help for rename
+      --name string           Channel Name
```

Comment:
Let's move this above help.

Reply:
this is autogenerated, i didnt do it manually

Reply:
Oh it's alphabetical. Got it.